src/AspNetCoreModuleV2/RequestHandlerLib/AppOfflineTrackingApplication.cpp

```diff
@@ -26,11 +24,9 @@ HRESULT AppOfflineTrackingApplication::StartMonitoringAppOffline()
     return hr;
 }
 
-void AppOfflineTrackingApplication::Stop(bool fServerInitiated)
+void AppOfflineTrackingApplication::StopInternal(bool fServerInitiated)
 {
-    APPLICATION::Stop(fServerInitiated);
-
-    m_status = APPLICATION_STATUS::RECYCLED;
+    APPLICATION::StopInternal(fServerInitiated);
```

Comment:
Should `StopInternal` be called at the end of this function?

Reply:
Why?

Reply:
Nvrm I miss read something the application status. We always want to call the base class before executing our own function. Ex calling `APPLICATION::StopInternal()` before anything we need to do in AppOfflineTrackingApplication.